docs/release-management.md

```diff
@@ -0,0 +1,159 @@
+        by the administrator.
+
+    components:
+      - submariner: <commit-id> 
+      - lighthouse: <commit-id>
+      - submariner-operator: <commit-id>
+      - submariner-charts: <commit-id>
+      - shipyard: <commit-id>
+      - admiral: <commit-id>
+
+   ```
+
+## verification of the release yaml
+
+The verification step would:
+* Verify the format of the yaml file, it should be valid yaml and have the required
+  fields to execute the release.
+
+* Verify that passed CI runs exist for the commit-ids proposed for the components,
+  we may have another yaml file in the releases repo defining the jobs that we consider
+  stable, as sometimes we have less stable jobs because of new features.
+
+* Verify that submariner-operator dependencies to projects in go.mod are pointing
+  to the proposed submariner, lighthouse, and shipyard releases, because this affects
+  the behaviour of `subctl verify` to be up-to-date with the e2e tests of those repositories.
+
+* Verify that projects have Dockerfile.dapper pinned to the specific shipyard release.
+
+* Verify that submariner/lighthouse are using the specific admiral commit ID, otherwise stop.
+
+## merge of the release yaml
+
+* A script will run in Github actions, executing step by step the tagging and verification
+of the multiple components.
+
+It will retry builds if necessary.
+
+* Shipyard & Admiral will be tagged for reference
+
+* The release notes will be added to the website, proposed as a PR.
+
+* Shipyard Dockerfile.dapper will be unpinned from all repositories in master via PR.
+
+## Handling of versions.go in subctl/operator
+
+Subctl and the operator have a set of version references about the other project they manage,
+this is handled by updating a versions.go in the repository. For this management strategy
+this will need to change in the following way:
+
+### "devel" reference
+
+versions.go will reference devel by default in the source code, but will also work
+as a placeholder for anybody compiling subctl with:
+
+* `-X github.com/submariner-io/submariner-operator/pkg/versions.LighthouseVersion=devel`
+* `-X github.com/submariner-io/submariner-operator/pkg/versions.SubmarinerVersion=devel`
+* `-X github.com/submariner-io/submariner-operator/pkg/versions.OperatorVersion=devel`
+
+### references pinned at compilation
+
+#### For a master build
+
+For the [devel bleeding edge](https://github.com/submariner-io/submariner-operator/releases/tag/devel)
+release channel, every component will be pinned at compilation time to the last successfully
+published image of each component (race conditions apart we assume this is what we tested
+in E2E). 
+
+* `-X github.com/submariner-io/submariner-operator/pkg/versions.LighthouseVersion=${LH_IMAGE_ID}`
+* `-X github.com/submariner-io/submariner-operator/pkg/versions.SubmarinerVersion=${SUBM_IMAGE_ID}`
+* `-X github.com/submariner-io/submariner-operator/pkg/versions.OperatorVersion=${OPERATOR_IMAGE_ID}`
```

Comment:
We won't have control over the specific versions anymore, so I say let's use just "devel" here.